app/src/main/java/pm/gnosis/heimdall/data/preferences/PreferencesWalletConnect.kt

```diff
@@ -0,0 +1,46 @@
+package pm.gnosis.heimdall.data.preferences
+
+import android.content.Context
+import android.content.SharedPreferences
+import pm.gnosis.model.Solidity
+import pm.gnosis.utils.asEthereumAddress
+import pm.gnosis.svalinn.common.utils.edit
+import pm.gnosis.utils.asEthereumAddressString
+
+class PreferencesWalletConnect(context: Context) {
+
+    private val bridgePreferences: SharedPreferences
+    private val sessionsPreferences: SharedPreferences
+
+    init {
+        bridgePreferences = context.getSharedPreferences(PREFERENCES_BRIDGE, Context.MODE_PRIVATE)
+        sessionsPreferences = context.getSharedPreferences(PREFERENCES_SESSIONS, Context.MODE_PRIVATE)
+    }
+
+    var keyIntroDone: Boolean
```

Comment:
variable name is misleading ... would just vall it `introDone` since this variable is not a key

Reply:
yes